solidity/dashboard/src/pages/TBTCApplicationPage.jsx

```diff
@@ -139,20 +139,47 @@ const TBTCApplicationPage = () => {
           title: "Success",
           content: "Authorization transaction successfully completed",
         })
+        setTimeout(() => onSuccessCallback(contractName, operatorAddress), 5000)
+      } catch (error) {
+        showMessage({
+          type: messageType.ERROR,
+          title: "Authorization action has failed ",
```

Comment:
not super strong on this, but what do you think about unifying notification messages for authorization/deathorization? maybe just simply:
- "Authorization successfully completed"
- "Authorization has failed"
- "Deauthorization successfully completed"
- "Deauthorization has failed"